core/dbt/contracts/graph/nodes.py

```diff
@@ -378,6 +378,10 @@ def same_contents(self, old, adapter_type) -> bool:
     def is_external_node(self):
         return False
 
+    @property
+    def all_constraints(self) -> List[Union[ModelLevelConstraint, ColumnLevelConstraint]]:
+        return []
```

Comment:
ditto

Reply:
Thinking about this more, I'm going to remove this. All existing references to `all_constraints` are behind isinstance checks so it shouldn't be necessary and does create a leaky abstraction where the base needs to know about its subclasses, bleugh